HHS/Head-Start-TTADP
# TTAHUB-313: Update Activity Report Landing to Use New Filter

## Description of change

Update activity report landing to use the new filter. 

The three components (Overview, Alerts, AR) on the landing page should now all respect a single filter at the top of the page.

When visiting the page thee filter should only set the region filter (based on the users permissions). 

It should not implement any date constraints by default.

## How to test

View the AR landing page. Adjusting the filter at the top should update all three components on the page. All existing page functionality like exporting reports and sorting should still work as it did before.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-313

## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
